src/components/MyAccount/profileTabs.test.tsx

```diff
@@ -1,23 +1,19 @@
 import React from "react"
-import { render, fireEvent, act } from "@testing-library/react"
+import { render } from "@testing-library/react"
 import ProfileTabs from "./ProfileTabs"
 import {
   mockCheckouts,
   mockFines,
   mockHolds,
   mockPatron,
 } from "../../../__test__/fixtures/accountFixtures"
-import { useRouter } from "next/router"
+import mockRouter from "next-router-mock"
+import userEvent from "@testing-library/user-event"
 
-jest.mock("next/router", () => ({
-  useRouter: jest.fn(),
-}))
+window.HTMLElement.prototype.scrollIntoView = jest.fn()
```

Comment:
Moved this into `jest.setup` on tabs PR, which makes this implementation of `mockRouter` unnecessary (can just use `fireEvent`)

Reply:
ok! i'll remove the mock here, but I think we should be testing the url behavior and not the router.push calls.

Reply:
Ok changed to your implementation